src/Umbraco.Infrastructure/HostedServices/ServerRegistration/InstructionProcessTask.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Umbraco.Core;
+using Umbraco.Core.Sync;
+
+namespace Umbraco.Infrastructure.HostedServices.ServerRegistration
+{
+    /// <summary>
+    /// Implements periodic database instruction processing as a hosted service.
+    /// </summary>
+    public class InstructionProcessTask : RecurringHostedServiceBase
+    {
+        private readonly IRuntimeState _runtimeState;
+        private readonly IDatabaseServerMessenger _messenger;
+        private readonly ILogger<InstructionProcessTask> _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InstructionProcessTask"/> class.
+        /// </summary>
+        public InstructionProcessTask(IRuntimeState runtimeState, IServerMessenger messenger, ILogger<InstructionProcessTask> logger)
+            : base(TimeSpan.FromSeconds(5), TimeSpan.FromMinutes(1))
```

Comment:
This was not hardcoded before (`_registrar.Options.RecurringSeconds*1000`)

Reply:
I think this effectively was, in that it came from `DatabaseServerMessengerOptions`, which currently is hard-coded.  But I think better anyway to move this to a configuration model, so if needed, it could be configured.  So I've done this.

For this one though I couldn't completely get rid of the original class - `DatabaseServerMessengerOptions` - as it also holds references to some callbacks.  Hence I've just renamed the class to `DatabaseServerMessengerCallbacks`, but perhaps some more refactoring there could be considered (i.e. does this need to be a separate class, or could the one property is has just be moved to `DatabaseServerMessenger`?